content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -358,7 +358,7 @@ EXECUTOR_NUMBER:: The unique number that identifies the current executor (among
 JAVA_HOME:: If your job is configured to use a specific JDK, this variable is set to the JAVA_HOME of the specified JDK. When this variable is set, PATH is also updated to include the bin subdirectory of JAVA_HOME
 JENKINS_URL:: Full URL of Jenkins, such as \https://example.com:port/jenkins/ (NOTE: only available if Jenkins URL set in "System Configuration")
 JOB_NAME:: Name of the project of this build, such as "foo" or "foo/bar".
-NODE_NAME:: The name of the node the current build is running on. Set to 'master' for master node.
+NODE_NAME:: The name of the node the current build is running on. Set to 'controller' for controller node.
```

This is exactly the correct transformation based on the instructions that I included, but now that I read it, I think the instructions should be changed.  The first executor agent available to Jenkins is on the controller.  It currently has the label 'master'.  Changing that label to 'controller' will cause a compatibility challenge for users with job definitions that depend on the label 'master'.

Sorry that I'm change the instructions now.  I propose the following change.  What do you think?
```suggestion
NODE_NAME:: The name of the node the current build is running on. Set to 'master' for the Jenkins controller.
```